app/embeds/index.js

```diff
@@ -148,6 +149,13 @@ export default [
     component: GoogleSlides,
     matcher: matcher(GoogleSlides),
   },
+  {
+    title: "Google Forms",
+    keywords: "survey",
+    icon: () => <Img src="/images/google-forms.png" />,
```

Comment:
Comment for icon itself – can you add a version that goes all the way to the top and bottom edges otherwise it will look smaller than the rest in the embed menu.